ADOPTERS.md

```diff
@@ -10,3 +10,4 @@ Below is a list of companies and projects that have adopted Open Cluster Managem
 - [RamenDR](https://github.com/RamenDR)
 - [Red Hat](https://www.redhat.com/)
 - [Xiao Hong Shu](https://www.xiaohongshu.com/)
+- [Andy Anderson](https://linkedin.com/in/clubanderson)
```

Comment:
Could you move this line up so that it meets the alphabetical order.

Reply:
done

Reply:
please approve